managed/src/main/java/com/yugabyte/yw/commissioner/tasks/subtasks/WaitForServer.java

```diff
@@ -55,7 +55,7 @@ public void run() {
       closeClient(client);
     }
     if (!ret) {
-      throw new RuntimeException(getName() + " did not respond to pings in the set time.");
+      throw new RuntimeException(getName() + " did not respond in the set time.");
```

Comment:
the ping here referred to a separate RPC called ping that exists on the db server. But agree that it can be confusing, accepted.